### DISCUSSION TITLE
My nvim config isn't applied

### DISCUSSION THREAD

**Discussion by J-Kappes** [mikesmithgh/kitty-scrollback.nvim#48] [Q&A] [answered]
Is this normal?

**Comment by mikesmithgh** [comment, marked as answer]
Hey @J-Kappes thank you for the question. When you say your config isn't applied do you mean your default configuration in `~/.config/nvim` ?

If so, by default kitty-scrollback.nvim runs nvim with the parameters `--clean --noplugin -n`. This runs nvim without a configuration, no plugins, and in memory.

You can change this behavior by configuring the `action_alias` or `map` commands in `kitty.conf`.

For example, this runs with the default arguments `--clean --noplugin -n`
```
action_alias kitty_scrollback_nvim kitten /Users/mike/gitrepos/kitty-scrollback.nvim/python/kitty_scrollback_nvim.py
```
- Equivalent to `nvim --clean --noplugin -n`

If you don't want to pass any arguments to nvim, you can use `--no-nvim-args`. This will run you default Neovim configuration.
```
action_alias kitty_scrollback_nvim kitten /Users/mike/gitrepos/kitty-scrollback.nvim/python/kitty_scrollback_nvim.py --no-nvim-args
```
- Equivalent to `nvim`


If you want to pass customized arguments to nvim, you can use `--nvim-args your-args-here`
```
action_alias kitty_scrollback_nvim kitten /Users/mike/gitrepos/kitty-scrollback.nvim/python/kitty_scrollback_nvim.py --nvim-args -u NONE +'lua vim.defer_fn(function() vim.cmd("ls!") end, 1000)'
```
this is just an example to show how you can pass more arguments.
- Equivalent to `nvim -u NONE +'lua vim.defer_fn(function() vim.cmd("ls!") end, 1000)'`

**Reply by J-Kappes** [reply]
Ah, yes. I mean my default nvim config. I'll try `--no-nvim-args`. Thank you.
